docs/pages/linking/overview.mdx

```diff
@@ -0,0 +1,108 @@
+sidebar_title: Overview
+description: An overview of available resources to implement Linking and Deep Links in your Expo apps.
+---
+
+import { AndroidIcon } from '@expo/styleguide-icons/custom/AndroidIcon';
+import { AppleIcon } from '@expo/styleguide-icons/custom/AppleIcon';
+import { BookOpen02Icon } from '@expo/styleguide-icons/outline/BookOpen02Icon';
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { CODE } from '~/ui/components/Text';
+
+## Linking
+
+Linking allows your app to interact with incoming and outgoing URLs. In this process, the user not only gets directed to open your app, but they are taken to a specific screen (route) within the app.
+
+For example, by clicking an advertisement of your product, your app will open on the user's device and they can view that product's details. This product's link that the user clicked may look like:
+
+```html
+myapp://web-app.com/product
+```
+
+This link is constructed by three parts:
+
+- **Scheme**: The URL scheme that identifies the app that should open the URL (example: `myapp://`). It can also be `https` or `http` for non-standard deep links.
+- **Host**: The domain name of the app that should open the URL (example: `web-app.com`).
+- **Path**: The path to the screen that should be opened (example: `/product`). If the path isn't specified, the user is taken to the home screen of the app.
+
+## Linking strategies
+
+There are different linking strategies you handle in your Expo app:
+
+- Linking to other apps from your app (outgoing links)
+- Linking to your app from other apps or a web browser (incoming links)
+- Android App Links
+- iOS Universal Links
+
+> **info** **Tip:** We recommend using [Development builds](/develop/development-builds/introduction/) to test your app's linking strategies instead of Expo Go.
+
+### Into other apps
+
+Linking to other apps from your app is achieved using a URL based on the target app's URL scheme. This **URL scheme** allows you to reference resources within that native app.
+
+Your app can use a [common URL scheme](/linking/into-other-apps/#common-url-schemes) for default apps or implement any URL scheme, including `https` and `http` (commonly used by web browsers like Chrome, Safari, and so on), and use JavaScript to handle the URL that launches the corresponding native app.
+
+<BoxLink
+  title="Linking into other apps"
+  description="Learn how to handle common and custom URL schemes to link other apps from your app."
+  href="/linking/into-other-apps/"
+  Icon={BookOpen02Icon}
+/>
+
+### Into your app
+
+Linking to your app from other apps or a web browser is achieved using a URL based on the scheme for your app. These URLs are called **deep links**.
+
+<BoxLink
+  title="Linking to your app"
+  description="Learn how to configure custom URL schemes to create a deep link of your app."
+  href="/linking/into-your-app/"
+  Icon={BookOpen02Icon}
+/>
+
+### Android App Links
```

Comment:
Can we group app links and universal links under a single heading, and then break out in much more brief into each platform? The story for each is basically the same: "This is how you route an http(s) link to a particular domain to your app if it is installed", just they're named differently per platform. I liken these to "App Shortcuts / Quick Actions": same concept that we'd probably call by the same name if Google and Apple could settle on one, and even then, we probably just pick the name from one platform and use it colloquially to refer to both.

Reply:
We used to use Universal links before this PR made a division. Do you want me to still use "Universal links"?

Reply:
Here's the update based on your initial feedback:

![CleanShot 2024-09-23 at 19 27 20](https://github.com/user-attachments/assets/14b9056f-b301-4f32-a891-91ec043221ab)